jbmc/unit/java_bytecode/ci_lazy_methods/lazy_load_lambdas.cpp

```diff
@@ -12,7 +12,7 @@ Author: Diffblue Limited.
 
 SCENARIO(
   "Lazy load lambda methods",
-  "[core][java_bytecode][ci_lazy_methods][lambdas][!mayfail]")
```

Comment:
`mayfail` doesn’t stop a test from running, it just means test failure isn’t counted.

Reply:
A test that can pass or fail is bad - since no body will observe if it changes state. I believe this has been erroneously marked as mayfail, since it passes.